sdk/cosmosdb/cosmos/test/internal/session.spec.ts

```diff
@@ -70,6 +74,95 @@ describe("New session token", function () {
   });
 });
 
+describe("Integrated Cache Staleness", async function (this: Suite) {
+  beforeEach(async function () {
+    await removeAllDatabases();
+  });
+  const dbId = addEntropy("maxIntegratedCacheTestDB");
+  const containerId = addEntropy("maxIntegratedCacheTestContainer");
+  const dedicatedGatewayMaxAge = 600000;
+  const client = new CosmosClient({
+    endpoint,
+    key: masterKey,
+    consistencyLevel: ConsistencyLevel.Eventual,
```

Comment:
What happens when the consistency is "session" or greater? Should the request fail in such case. Anyway, you do need at least a test case to catch this configuration.

Reply:
If the session level is missing or incorrect, this parameter is ignored. This catch is handled in the service.